cpp/dolfinx/fem/DirichletBC.cpp

```diff
@@ -409,7 +408,7 @@ fem::locate_dofs_topological(const fem::FunctionSpace& V, const int dim,
   const int num_entity_closure_dofs
       = dofmap->element_dof_layout->num_entity_closure_dofs(dim);
   std::vector<std::int32_t> dofs;
-  for (Eigen::Index i = 0; i < entities.rows(); ++i)
+  for (size_t i = 0; i < entities.size(); ++i)
```

Comment:
I think the C ++ version `std :: size_t` is preferable.

Reply:
Of course, that was unintentional :D
Thanks!